src/packer.pkr.hcl

```
@@ -83,51 +91,48 @@ source "amazon-ebs" "example" {
   region             = var.build_region
   region_kms_key_ids = var.region_kms_keys
   skip_create_ami    = var.skip_create_ami
-  source_ami         = data.amazon-ami.debian_bullseye.id
-  ssh_username       = "admin"
+  source_ami         = data.amazon-ami.windows_server_2022.id
   subnet_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
   tags = {
-    Application        = "Example"
-    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    Application        = "Windows Server 2022"
```

We should use this tag to distinguish between AMIs created with this repo and those created by `cisagov/windows-server-packer`:
```suggestion
    Application        = "Windows Commando VM"
```